definitions/checks/sca_only.rb

```diff
@@ -0,0 +1,57 @@
+class Checks::CheckOrganizationContentAccessMode < ForemanMaintain::Check
+  metadata do
+    label :check_organization_content_access_mode
+    description 'Check if any organizations are using entitlement mode'
+    tags :pre_upgrade
+
+    confine do
+      feature(:katello)
+    end
+  end
+
+  def run
+    if feature(:instance).downstream
+      versioned_instance_name = "Satellite 6.16"
+      doc_link = "https://access.redhat.com/articles/simple-content-access"
+    else
+      versioned_instance_name = "Katello 4.12"
+      doc_link = "https://theforeman.org/plugins/katello/nightly/user_guide/simple_content_access/index.html"
+    end
+
+    with_spinner('Checking organization content access modes') do
+      output = execute_org_check
+      orgs_to_migrate = parse_orgs_to_migrate(output)
+
+      assert(
+        orgs_to_migrate.empty?,
+        "The following organizations are using entitlement mode:\n"\
+        "#{orgs_to_migrate.join(', ')}\n"\
+        "As of #{versioned_instance_name}, entitlement mode is removed, and these organizations"\
+        " will be automatically migrated to Simple Content Access during the upgrade."\
+        " Please ensure that you have reviewed the documentation and are prepared for this change."\
+        " Documentation for SCA mode is available at #{doc_link}"
+      )
+    end
+  end
+
+  private
+
+  # rubocop:disable Lint/InterpolationCheck
+  def execute_org_check
+    execute!(
+      'echo "Organization.all.each ' \
+      '{ |org| puts \"NONSCA: #{org.name}\" unless org.simple_content_access? }" ' \
+      '| foreman-rake console'
+    )
```

Comment:
Are we able to use our API here? I suppose this is a one off check so it doesn't matter as much for a longer-term support.

Reply:
Well, I actually broke the damn thing again while trying to appease rubocop:

```
The following organizations are using entitlement mode:
#{org.name}" unless org.simple_content_access? }, Default Organization
```

I'd still rather fix that up and also test that it works properly with multiple SCA org and no SCA orgs, but could pivot to API if it ends up being more trouble than it's worth.